gauge/tools/chocolateyInstall.ps1.template

```diff
@@ -8,4 +8,9 @@ Write-Host -ForegroundColor yellow "We are constantly looking to make Gauge bett
 
 Install-ChocolateyZipPackage -PackageName "$packageName" -Url "$url" -Url64 "$url64" -UnzipLocation "$location" -validExitCodes $validExitCodes
 
-setx /m path "$env:PATH;$location"
+$currentPath = (Get-Itemproperty -path 'hklm:\system\currentcontrolset\control\session manager\environment' -Name Path).Path
+
+if($currentPath -NotLike "*$location*") {
+    $newPath = $currentPath + "$location"
```

Comment:
the `PATH` values are semi-colon (`;`) separated. It should be included in the appended string..

Also consider using something like [this](https://stackoverflow.com/a/2571200/326543) instead of querying the registry.